osh.spec

```
@@ -156,23 +156,32 @@ OpenScanHub worker devel configurations.
 
 %build
 
-# Add -s to the shebang in osh/client/osh-cli:
-# https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_shebang_macros
-# TODO: Remove this when we migrate to newer Python packaging macros which
-# do this automatically.
-%py3_shebang_fix osh/client/osh-cli
-
 # collect static files from Django itself
 PYTHONPATH=. osh/hub/manage.py collectstatic --noinput
 
 # set path to python sitelib in the example httpd config
 sed 's|@PYTHON3_SITELIB@|%{python3_sitelib}|' osh/hub/osh-hub-httpd.conf.in > osh/hub/osh-hub-httpd.conf
 
+%if 0%{?fedora} > 41
```

Do you also build this package on RHEL/EPEL? If so, please note that this will likely not work in RHEL 11+.

You would need:

```suggestion
%if 0%{?fedora} > 41 || 0%{?rhel} > 10
```

Or rhel > 8.

---

Fixed.